9.2/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -114,6 +114,19 @@ function generate_postgresql_recovery_config() {
       > "${POSTGRESQL_RECOVERY_FILE}"
 }
 
+# this function iterates over hooks.d and seek for folders
+# which contains post-init executables
+function execture_post_init_scripts() {
+  for i in ${CONTAINER_SCRIPTS_PATH}/hooks.d/*
```

Comment:
Like @praiskup wrote it's default value which is defined in Dockerfile.

I will rename it then. However idea behind hooks.d was that there can be located also post-init or pre-init tasks in future.